src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -184,6 +187,11 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
                     for (const project of response.body.user.projects) {
                         this.usersProjects.push(project);
                     }
+
+                    // notifies the user that he/she is not part of any project
+                    if (this.usersProjects.length === 0) {
+                        this.errorMessage = 'You are not part of any activated project.';
```

Comment:
'You are not a part of any active projects.'

Reply:
fixed in 3717fc5